batch/sql/estimated-current.sql

```diff
@@ -229,6 +229,8 @@ CREATE TABLE IF NOT EXISTS `batch_updates` (
   `token` VARCHAR(100) DEFAULT NULL,
   `start_job_id` INT NOT NULL,
   `n_jobs` INT NOT NULL,
+  `start_job_group_id` INT NOT NULL DEFAULT 0,
+  `n_job_groups` INT NOT NULL DEFAULT 1,
```

Comment:
Can you explain why it's necessary for QoB? That might inform whether I think it's good to go now or later.

Reply:
You're right for QoB that each update will only have one job group at a time in an update. However, we'll eventually want to support being able to add multiple job groups at a time in a single update. If we don't add this now, then we'll need to do a migration later on to add these fields when we're ready to add multiple job groups.

Note that this decision and the reasoning for simplifying it is going to change #14011 and the scope of that change. If that's the case, then I need to pivot ASAP as that code already puts in all of the changes necessary. So maybe take a look at that PR first (still fixing some issues, but QoB tests are passing) because that is going to dictate what is in here.

Reply:
I think the answer to what to put in #14011 and the MVP is going to dictate what to do in the other two places.